cocos/audio/audio-source.ts

```diff
@@ -197,6 +199,30 @@ export class AudioSource extends Component {
         return this._volume;
     }
 
+    /**
+     * @en
+     * The playback rate of this audio source (0.0 to 10.0).<br>
+     * Note: Playback rate may be ineffective on some platforms.
+     * @zh
+     * 此音频源的播放速率（0.0 到 10.0）。<br>
+     * 注意：播放速率在某些平台上可能无效。<br>
+     */
+    @range([0.0, 10.0])
+    @tooltip('i18n:audio.playbackrate')
+    set playbackRate (val: number) {
+        if (Number.isNaN(val)) { console.warn('illegal playback rate!'); return; }
```

Comment:
We don’t want to help developers verify whether this value is `NaN`
__TypeScript__ should tell developers that they are passing an `NaN` to the `playbackRate` property,  which is a wrong approach

We hope to use the language to do the type checking work